src/Core/Checkout/Order/Aggregate/OrderTransactionCaptureRefund/OrderTransactionCaptureRefundDefinition.php

```diff
@@ -59,7 +62,9 @@ protected function defineFields(): FieldCollection
 
         return new FieldCollection([
             (new IdField('id', 'id'))->addFlags(new ApiAware(), new PrimaryKey(), new Required()),
+            (new VersionField())->addFlags(new ApiAware()),
             (new FkField('capture_id', 'captureId', OrderTransactionCaptureDefinition::class))->addFlags(new ApiAware(), new Required()),
+            (new ReferenceVersionField(OrderTransactionDefinition::class, 'capture_version_id'))->addFlags(new ApiAware(), new Required()),
```

Comment:
```suggestion
            (new ReferenceVersionField(OrderTransactionCaptureDefinition::class, 'capture_version_id'))->addFlags(new ApiAware(), new Required()),
```

please run `bin/console dal:validate` to check DAL

Reply:
thanks! fixed, `dal:validate` does not report anything else, so this should be all good now